statsmodels/base/data.py

```diff
@@ -365,7 +367,12 @@ def _get_row_labels(self, arr):
 
     def _get_names(self, arr):
         if isinstance(arr, DataFrame):
-            return list(arr.columns)
+            if isinstance(arr.columns, MultiIndex):
+                # Flatten MultiIndexes into "simple" column names
+                return ['_'.join((level for level in c if level))
+                        for c in arr.columns]
+            else:
+                return list(arr.columns)
```

Comment:
I think there is a (fairly new) MUktiIndex method for this.  Oventuakly we should just use that